pkg/visor/visorconfig/values.go

```diff
@@ -180,12 +180,23 @@ func DmsgPtyWhiteList() string {
 func MustPK(pk string) cipher.PubKey {
 	var sPK cipher.PubKey
 	if err := sPK.UnmarshalText([]byte(pk)); err != nil {
+		fmt.Printf("invalid public key: %s", pk)
 		panic(err)
 	}
 
 	return sPK
 }
 
+// MustPKs unmarshals comma separated list of string PKs to []cipher.PubKey. It panics if unmarshaling fails.
+func MustPKs(pks string) []cipher.PubKey {
```

Comment:
Can we name this more explicitly like `MustParsePKs`?

Reply:
If we want to rename the one above it to match as `MustParsePK`